### DISCUSSION TITLE
preprocessing and validation

### DISCUSSION THREAD

**Discussion by ritu111** [pycaret/pycaret#2013] [Q&A] [answered]
I have trained classification model(on 80% data) which gives auc of 0.63 on kfold cross validation using pycaret. And saved the pipeline, and tested model pipeline on remaining 20% data, which gives auc of 0.5. Not sure why the results on separate validation set are low. 
Because 0.63 AUC is also test accuracy which is returned after training model. But when I am trying to calculate AUC separately, AUC is reducing considerably.

**Comment by ritu111** [comment, marked as answer]
Solved this issue by passing raw_score = True in predict_model function, and then we get two columns Score_0 and Score_1. By using Score_1 to calculate auc fixed the issue

**Reply by rahulsahai78** [reply]
How did you use the Score_1 to calculate the auc?
